### PR TITLE
Update develop-client-apps.md

### PR DESCRIPTION
**Pull request by gewarren**
Added two words that were missing in the top paragraph.

### CODE REVIEW

File: docs/framework/develop-client-apps.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ ms.author: dotnetcontent
 manager: "wpickett"
 ---
 # Developing Client Applications with the .NET Framework
-There are multiple ways to develop Windows-based applications with the .NET framework that run locally on users' computers or devices. This section contains topics that describe how to create Windows-based applications by using Windows Presentation Foundation (WPF) or by using Windows Forms. However, you also create client applications using the .NET Framework and make them available to your users through the Windows Store and Windows Phone store for computers or devices, web applications that can be viewed in a browser.  
+There are multiple ways to develop Windows-based applications with the .NET framework that run locally on users' computers or devices. This section contains topics that describe how to create Windows-based applications by using Windows Presentation Foundation (WPF) or by using Windows Forms. However, you can also create client applications using the .NET Framework and make them available to your users through the Windows Store and Windows Phone store for computers or devices, and web applications that can be viewed in a browser.
```
Comment (rpetrusha) [inline docs/framework/develop-client-apps.md]: for computers or devices, and web applications --> for computers, devices, and web applications

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @gewarren. I've made one suggested change.

File: docs/framework/develop-client-apps.md (hunk 1 above)
Comment (gewarren) [reply docs/framework/develop-client-apps.md]: Thanks Ron. The only thing is that makes it sounds like web applications are available through the Windows Store and Windows Phone Store. How about:

However, you can also create web applications using the .NET Framework, and client applications  for computers or devices that you make available through the Windows Store or Windows Phone Store.

Comment (rpetrusha) [reply docs/framework/develop-client-apps.md]: That sounds good, @gewarren.

**Comment by gewarren** [comment]
@rpetrusha I made the changes we agreed on. I'm not sure how to resolve the "changes requested" blocker. Maybe you can? Thanks for your help.

**Comment by rpetrusha** [comment]
@gewarren, "changes requested" is removed either when the person who requested changes approves them or dismisses the review, or when someone else with the necessary permissions dismisses the review that requests changes. This looks good, so I'll approve and merge.

**Review by rpetrusha — APPROVED** [review]

**Comment by gewarren** [comment]
Thanks Ron!

**Pull request merged**